### PR TITLE
Pull my finger again

### PR DESCRIPTION
**Pull request by Saulis**
[Review on Reviewable](https://reviewable.io/reviews/saulis/components-1/9)

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="http://app.review.ninja/Saulis/components-1/pull/9" target="_blank">ReviewNinja

File: pom.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <version>0.2.0</version>
 
     <properties>
-      <gwtversion>2.8.0-SNAPSHOT</gwtversion>
+      <gwtversion>2.8.0.VC-SNAPSHOT</gwtversion>
```
Comment (Saulis) [inline pom.xml]: VC?

Comment (Saulis) [inline pom.xml]: YES

**Comment by Saulis** [comment]
Review status: 0 of 2 files reviewed, 3 unresolved discussions, some commit checks failed.

---

<sup>**[vaadin-components-gwt/src/main/webapp/vaadin-grid/vaadin-grid.html, line 10 [r1]](https://reviewable.io:443/reviews/saulis/components-1/9#-Jpaah5c57WMvp7wrStM)** ([raw file](https://github.com/saulis/components-1/blob/42727d35cffb3e93432fec622db897686b4899c4/vaadin-components-gwt/src/main/webapp/vaadin-grid/vaadin-grid.html#L10)):</sup>
El comment

---

<sup>**[vaadin-components-gwt/src/main/webapp/vaadin-grid/vaadin-grid.html, line 22 [r1]](https://reviewable.io:443/reviews/saulis/components-1/9#-Jpaak3lPBgr2lRtDwpy)** ([raw file](https://github.com/saulis/components-1/blob/42727d35cffb3e93432fec622db897686b4899c4/vaadin-components-gwt/src/main/webapp/vaadin-grid/vaadin-grid.html#L22)):</sup>
Multiple comments

---

Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/saulis/components-1/9)

**Comment by Saulis** [comment]
Review status: all files reviewed, all discussions resolved, some commit checks failed.
Reviewed files:
- pom.xml @ r1
- vaadin-components-gwt/src/main/webapp/vaadin-grid/vaadin-grid.html @ r1

---

Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/saulis/components-1/9)

**Pull request closed without merging**